### PR TITLE
Fix blueprint tests

### PR DESCRIPTION
**Pull request by jkozol**

### CODE REVIEW

File: test/verify/check-blueprint [hunk 1]
```diff
@@ -149,8 +141,8 @@ class TestBlueprint(composerlib.ComposerCase):
         b.wait_visible("#cmpsr-modal-export")
         with b.wait_timeout(300):
             b.wait_in_text("#textInput2-modal-markup", "openssh-server")
```
Comment (lucasgarfield) [inline test/verify/check-blueprint:143]: I'm getting a timeout failure here.

File: components/Toolbar/BlueprintsToolbar.js [hunk 2]
```diff
@@ -1,79 +1,77 @@
...
+          <SearchInput
+            aria-label="blueprints search input"
+            onChange={onInputChange}
+            value={inputValue}
+            onClear={() => onInputChange("")}
+          />
+        </ToolbarItem>
+        <ToolbarItem variant="icon-button-group">
+          {(props.sortValue === "DESC" && (
+            <Button variant="plain" aria-label="sort asc" onClick={() => props.sortSetValue("ASC")}>
```
Comment (lucasgarfield) [inline components/Toolbar/BlueprintsToolbar.js:35]: Would it be better for screen readers if the aria label was `sort ascending`?

File: components/Toolbar/BlueprintsToolbar.js [hunk 3]
```diff
@@ -1,79 +1,77 @@
...
+              <SortAlphaDownIcon />
+            </Button>
+          )) ||
+            (props.sortValue === "ASC" && (
+              <Button variant="plain" aria-label="sort desc" onClick={() => props.sortSetValue("DESC")}>
```
Comment (lucasgarfield) [inline components/Toolbar/BlueprintsToolbar.js:40]: Same as above, `sort descending` might be better for screen readers...

File: components/Toolbar/BlueprintsToolbar.js (hunk 2 above)
Comment (jkozol) [reply components/Toolbar/BlueprintsToolbar.js:35]: Good point!

File: components/Toolbar/BlueprintsToolbar.js (hunk 3 above)
Comment (jkozol) [reply components/Toolbar/BlueprintsToolbar.js:40]: Good point!

**Pull request merged**